examples/rllib/utils.py

```diff
@@ -95,6 +96,33 @@ def get_dmlab2d_env(self):
     """Returns the underlying DM Lab2D environment."""
     return self._env
 
+  metadata = {'render.modes': ['rgb_array']}
```

Comment:
This doesn't seem to be used anywhere here. If needed, please name this `METADATA` as it is a module level constant and use immutabledict as the outer type, and use a tuple instead of a list to avoid being able to accidentally mutate this object at runtime.

Reply:
I see this might be a hard requirement for RLLib. If that's correct, please add a comment explicitly explaining this.

Reply:
I agree it's a bit unusual! The gym `Env` class that is being extended, includes this in the `render` docstring:

> Note: Make sure that your class's metadata 'render.modes' key includes the list of supported modes...

Alongside this example:

```python
metadata = {'render.modes': ['human', 'rgb_array']}
```